server/migrations/20201212153557-add-index-to-matchmaking-times.js

```diff
@@ -0,0 +1,15 @@
+exports.up = async function (db) {
+  await db.runSql(`
+    CREATE INDEX start_date_index ON matchmaking_times (start_date);
```

Comment:
I'm not sure this will actually be used for the query we will be running most often (the current matchmaking time). I think you'd need to store it in the reverse order for that (which is what we do for chat messages), e.g.

```sql
CREATE INDEX start_date_index ON matchmaking_times (start_date DESC);
```